backend/src/main/java/com/google/capstone/UploadImg.java

```diff
@@ -50,14 +49,18 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
   }
   
   /**
-   * Currently useless other than I need a doPost so the upload page isn't an error (not sure why yet)
+   * doesn't do anything other than handle the redirect from the backend back to the frontend (as of now)
    */
   @Override
   protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
       BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
 
       Map<String, List<FileInfo>> uploads = blobstore.getFileInfos(request);
       List<FileInfo> fileInfos = uploads.get("files");
-  }
 
+      //deployment link
+      //response.sendRedirect("https://igunda-isangimino-nstroupe.uc.r.appspot.com/user-uploads-gallery");
+      //development link
+      response.sendRedirect("https://3001-ba659410-163c-49e0-b45f-c22c5b2dc8b5.us-central1.cloudshell.dev/user-uploads-gallery");
```

Comment:
not sure about this absolute path. I'm worried that it will stop working if the hostname changes.